prow/plugins/lgtm/lgtm.go

```diff
@@ -60,9 +60,36 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	configInfo := map[string]string{}
+	for _, orgRepo := range enabledRepos {
+		parts := strings.Split(orgRepo, "/")
+		if len(parts) != 2 {
```

Comment:
Can't you specify a config for the whole org, not just a repo? Aka I'd expect "kubernetes" or "kubernetes/test-infra" to be valid, but not "kubernetes/test-infra/master"

Reply:
Maybe, but in that case we've to correct it in:
- approve.go
- blockade.go
- slackevents.go
- trigger.go
- welcome.go

@cjwagner what do you think, as it looks like you were the first to [commit](https://github.com/kubernetes/test-infra/commit/66bf77506451ef37116557491d375328264bff55) this verification?

Reply:
Let's do it only in this file in this PR and make a follow-up to do it everywhere after.

Reply:
Alright!
So simply a `>` should do it, right?